README.md

```diff
@@ -1,5 +1,17 @@
 # bp2022-ap1
 
+## CI & CD
+
+`dev` branch: ![CI](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml/badge.svg?branch=dev)
+
+`main` branch: ![CI](https://github.com/BP2022-AP1/bp2022-ap1/actions/workflows/python-app.yml/badge.svg?branch=main)
```

Comment:
The workflow can be wrapped into a link as well. Would you mind doing that? (At least link to the actions page)

Reply:
Is also added